modules/core/src/main/java/org/apache/ignite/internal/processors/authentication/IgniteAuthenticationProcessor.java

```diff
@@ -677,8 +635,6 @@ private ClusterNode coordinator() {
                     U.warn(log, "Cannot find the server coordinator node. "
                         + "Possible a client is started with forceServerMode=true. " +
                         "Security warning: user authentication will be disabled on the client.");
-
-                    isEnabled = false;
```

Comment:
I see there 2 possible issues:
1. a misleading comment on the previous line, as user authentication isn't disabled now explicitly;
2. as there no check for `isEnabled` in ioLsnr then for the case described in the warning below (client is in forceServerMode=true, cfg.isAuthenticationEnabled=true and there no other nodes) we can hang on this chain:

`ioLsnr -> onAuthenticateRequestMessage -> authenticateOnServer -> readyForAuthFut.get();`

Also for `discoLsnr` there can be NPE for `onNodeLeft -> coordinator().id()`.

Reply:
1. I have fixed comment.
2. 
Mentioned by you problem is related only to client node with no server nodes are alive.
So onAuthenticateRequestMessage or onNodeLeft can't be called - they are require at leas one server node to be alive.
The problem appears during authentication method execution:
previously if no server node is found security was tuned off on client node and all authentication or user operations were skipped but this client node can't join the cluster with authentication enabled.
Now we can't turn off security so I propose to explicitly throw exception during authentication if no server nodes are alive.

Reply:
OK, thanks!